3scaleAdapter/cmd/main.go

```diff
@@ -79,6 +82,28 @@ func parseMetricsConfig() *metrics.Reporter {
 	return metrics.NewMetricsReporter(true, port)
 }
 
+func cacheConfigBuilder() *threescale.ProxyConfigCache {
+	cacheTTL := threescale.DefaultCacheTTL
+	cacheRefreshInterval := threescale.DefaultCacheRefreshBuffer
+	cacheEntriesMax := threescale.DefaultCacheLimit
+
+	if viper.IsSet("cache_ttl_seconds") {
+		ttl := time.Duration(viper.GetInt("cache_ttl_seconds"))
+		cacheTTL = time.Duration(time.Second * ttl)
+	}
+
+	if viper.IsSet("cache_refresh_seconds") {
+		refreshInterval := time.Duration(viper.GetInt("cache_refresh_seconds"))
+		cacheRefreshInterval = time.Duration(time.Second * refreshInterval)
+	}
+
+	if viper.IsSet("cache_entries_max") {
```

Comment:
I think we need to write down somewhere the collection of variables we can use to affect behaviour here.

Reply:
Yes absolutely. Covered in #39